README.md

```diff
@@ -74,6 +74,19 @@ This will run the app in archive-only mode and exit, creating gzipped bundles as
 
 Where `1655340486` is the timestamp, and `4.15.10` the site version.
 
+## Docker
+
+To build the docker image execute:
+
+    docker build . -t blofetcha
+
+To run the Docker image created above, execute:
+
+    docker run -e DISABLE_CHROME_SANDBOX=true -v /home/usre/code/blofetcha/docker-archive:/archive blofetcha
+
+NOTE: The mounted "docker-archive" volume will contain the archived Javascript files
+NOTE2: Currently the files are output as having `root:root` permissions and need to be changed after
```

Comment:
I would remove this line if it isn't something we can do about, and if it is we should just do it (?) so either way let's get rid of it, and then remove the `NOTE:` prefix in the previous one <\/nit> :)